workshops/templates/workshops/workshop_staff.html

```diff
@@ -13,21 +13,25 @@
   {% if persons %}
     <table class="table table-striped">
       <tr>
-	<th>DC?</th>
-	<th>SWC?</th>
+        <th>instr. badges</th>
```

Comment:
I can understand that, Greg, but I had to switch to other way of checking badges since it was causing Django to count wrong the "taught" column.